lib/ffi-vix_disk_lib/api.rb

```diff
@@ -18,7 +18,7 @@ def self.load_error
       #
       # Make sure we load one and only one version of VixDiskLib
       #
-      version_load_order = %w( 5.5.2 5.5.1 5.5.0 5.1.3 5.1.2 5.1.1 5.1.0 5.0.4 5.0.0 1.2.0 1.1.2 )
+      version_load_order = %w( 6.0.0 5.5.4 5.5.2 5.5.1 5.5.0 5.1.3 5.1.2 5.1.1 5.1.0 5.0.4 5.0.0 1.2.0 1.1.2 )
```

Comment:
Tested with 5.5.4, 5.5.1, and 6.0.0, against ESX 6.0, ESX 4.1, Vcenter 6.0, and Vcenter 4.1